Dockerfile

```diff
@@ -47,6 +47,7 @@ RUN apt-get -yq install gtk-chtheme ubuntu-mate-themes fonts-roboto
 RUN apt-get -yq install \
     gettext \
     build-essential \
-    cmake
+    cmake \
+    uuid-dev
```

Comment:
Looking at https://circleci.com/gh/BerryFarm/berrymuch/16#tests/containers/0 the sequence was fine and nothing got skipped.

You shouldn't need uuid-dev here because it's a compiled library from ports - not a host build dependency. Can you please remove it from the container and try again?